watson_developer_cloud/language_translator_v2.py

```diff
@@ -11,78 +13,812 @@
+        Identifies the language of the input text.
 
-    def get_identifiable_languages(self):
-        return self.request(method='GET', url='/v2/identifiable_languages',
-                            accept_json=True)
+        :param str text: Input text in UTF-8 format.
+        :return: A `dict` containing the `IdentifiedLanguages` response.
+        :rtype: dict
+        """
+        if text is None:
+            raise ValueError('text must be provided')
+        data = text
+        headers = {'content-type': 'text/plain'}
+        response = self.request(
+            method='POST',
+            url='/v2/identify',
+            headers=headers,
+            data=data,
+            accept_json=True)
+        return response
 
-    def get_models(self, default=None, source=None, target=None):
+    def list_identifiable_languages(self):
         """
-        Get the available models for translation
+        Lists all languages that can be identified by the API.
+
+        Lists all languages that the service can identify. Returns the two-letter code
+        (for example, `en` for English or `es` for Spanish) and name of each language.
+
+        :return: A `dict` containing the `IdentifiableLanguages` response.
+        :rtype: dict
         """
-        params = {'default': default, 'source': source, 'target': target}
-        return self.request(method='GET', url='/v2/models', params=params,
-                            accept_json=True)
+        response = self.request(
+            method='GET', url='/v2/identifiable_languages', accept_json=True)
+        return response
+
+    #########################
+    # models
+    #########################
 
-    def create_model(self, base_model_id, name=None, forced_glossary=None,
+    def create_model(self,
+                     base_model_id,
+                     name=None,
+                     forced_glossary=None,
                      parallel_corpus=None,
-                     monolingual_corpus=None):
-        if forced_glossary is None and parallel_corpus is None and \
-                        monolingual_corpus is None:
-            raise WatsonInvalidArgument(
-                'A glossary or corpus must be provided')
-        params = {'name': name,
-                  'base_model_id': base_model_id}
-        files = {'forced_glossary': forced_glossary,
-                 'parallel_corpus': parallel_corpus,
-                 'monolingual_corpus': monolingual_corpus}
-        return self.request(method='POST', url='/v2/models', params=params,
-                            files=files, accept_json=True)
+                     monolingual_corpus=None,
+                     forced_glossary_content_type=None,
+                     forced_glossary_filename=None,
+                     parallel_corpus_content_type=None,
+                     parallel_corpus_filename=None,
+                     monolingual_corpus_content_type=None,
+                     monolingual_corpus_filename=None):
```

Comment:
I think we probably do not want the `content_types` or `filename` parameters here.

Reply:
I see why this happened -- a typo in the annotations in the LT swagger.